tests/system.py

```diff
@@ -1599,10 +1601,60 @@ def test_dbapi_fetch_w_bqstorage_client_small_result_set(self):
         self.assertEqual(fetched_data, expected_data)
 
     @unittest.skipIf(
-        bigquery_storage_v1beta1 is None, "Requires `google-cloud-bigquery-storage`"
+        bigquery_storage_v1 is None, "Requires `google-cloud-bigquery-storage`"
     )
     @unittest.skipIf(fastavro is None, "Requires `fastavro`")
     def test_dbapi_fetch_w_bqstorage_client_large_result_set(self):
+        bqstorage_client = bigquery_storage_v1.BigQueryReadClient(
+            credentials=Config.CLIENT._credentials
+        )
+        cursor = dbapi.connect(Config.CLIENT, bqstorage_client).cursor()
+
+        # Pick a large enouhg LIMIT value to assure that the fallback to the
+        # default client is not needed due to the result set being too small
+        # (a known issue that causes problems when reding such result sets with
```

Comment:
So, the fix for small result sets is rolling out at we speak, and we shouldn't need to fallback in this case. We can leave this in here for now and clean it up when it's rolled out.

Reply:
Noted. Will create a ticket to clean it up when ready.